sdk/servicebus/service-bus/src/core/messageSender.ts

```diff
@@ -22,17 +22,20 @@ import {
   RetryOperationType,
   Constants,
   delay,
-  MessagingError
+  MessagingError,
+  RetryOptions
 } from "@azure/core-amqp";
 import {
   ServiceBusMessage,
   toAmqpMessage,
-  getMessagePropertyTypeMismatchError
+  getMessagePropertyTypeMismatchError,
+  CreateBatchOptions
```

Comment:
Why does `CreateBatchOptions` live in `serviceBusMessage.ts` file? I would imagine the right place to be `messageSender.ts`

Reply:
Moved to the `models.ts`, the place where all other `Options` are present.